src/com/facebook/buck/go/GoBuckConfig.java

```diff
@@ -155,8 +166,10 @@ private Tool getGoTool(
     }
 
     CommandTool.Builder builder = new CommandTool.Builder(new HashedFileTool(toolPath.get()));
-    for (String arg : getFlags(extraFlagsConfigKey)) {
-      builder.addArg(arg);
+    if (!extraFlagsConfigKey.isEmpty()) {
```

Comment:
This is a private helper to the functions above. The packer doesn't accept any flags, so it doesn't have an extra flags config key. I can make it an optional, but it seemed less verbose to use an empty string.